workflow/config/fcst.yaml

```diff
@@ -119,6 +118,7 @@ config_fcst:
     export dnats={doc.fv3_gfs_settings.phy_dependent_var.dnats}
     export do_sat_adj="{tools.fort(doc.fv3_gfs_settings.phy_dependent_var.do_sat_adjust)}"
     export nwat={doc.fv3_gfs_settings.phy_dependent_var.nwat}
+    export dddmp={doc.fv3_gfs_settings.dddmp}
```

Comment:
@WalterKolczynski-NOAA ... I don't know if it was adding this line or the removal of the other lines or both that are needed for the dddmp to get set to the value that was set in the case file.  But as it was before, even though the case file said dddmp=.2, it was being set as .1